mappings/net/minecraft/client/gui/DrawableHelper.mapping

```diff
@@ -36,51 +40,52 @@ CLASS net/minecraft/class_332 net/minecraft/client/gui/DrawableHelper
 		ARG 5 v
 		ARG 6 uWidth
 		ARG 7 vHeight
-		ARG 8 texWidth
-		ARG 9 texHeight
+		ARG 8 textureWidth
+		ARG 9 textureHeight
 	METHOD method_25294 fill (IIIII)V
 		ARG 0 x1
 		ARG 1 y1
 		ARG 2 x2
 		ARG 3 y2
 		ARG 4 color
-	METHOD method_25295 innerBlit (IIIIIFFFF)V
-		ARG 0 xStart
-		ARG 1 xEnd
-		ARG 2 yStart
-		ARG 3 yEnd
+	METHOD method_25295 drawTexturedQuad (IIIIIFFFF)V
+		ARG 0 x1
```

Comment:
The old names here were a bit more helpful. I don think they were ideal tho, but better than `x1`.

Reply:
Don't think I have seen those x and xEnd anywhere except here before, also I don't think it doesn't really have to be the start and end of a texture.

Reply:
These parameters are not part of mojang map, but our improvisations. I feel they are good and have no need to change.

Reply:
alrighty